getting_started/developers/developers_cli.adoc

```diff
@@ -0,0 +1,189 @@
+= Getting Started for Developers: CLI
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+:description: The getting started experience for Developers, using the CLI.
+:keywords: getting started, developers, cli, templates
+
+toc::[]
+
+// tag::overview[]
+
+== Overview
+
+This getting started experience walks you through the simplest way to get a
+sample project up and running on OpenShift. There are a few different ways to
+launch images within a project, but this topic focuses on the quickest and
+easiest method.
+
+If this is the first part of the documentation you have read, and you are
+unfamiliar with the core concepts of OpenShift version 3, you might want to
+start by reading about
+link:../../whats_new/overview.html[what's new]. This version
+of OpenShift is significantly different from version 2.
+
+The following sections guide you through creating a project that contains a
+sample NodeJS application that will serve a welcome page and the current hit
+count (stored in a database) to http://host:8080 .
+
+// end::overview[]
+
+// tag::beforeyoubegin[]
+
+== Before You Begin
+
+Before you can get started using OpenShift Enterprise:
+
+- You must be able to access a running instance of OpenShift Enterprise. If you do not have access, then contact your system administrator.
+- Your instance must be pre-configured by a system administrator with sample templates and builder images.
```

Comment:
@adellape -- do you know how I can point to the correct templates and builder images that are in use on the stage console?